eng/SourceBuildPrebuiltBaseline.xml

```diff
@@ -1,5 +1,5 @@
 <UsageData>
   <IgnorePatterns>
-    <UsagePattern IdentityGlob="*/*" />
+    <UsagePattern IdentityGlob="Microsoft.SourceBuild.Intermediate.*/*" />
```

Comment:
I don't think we should be enabling prebuilt detection with these changes.  There are likely other changes needed to get the repo level prebuilts clean.  We don't want to block this PR on them.